orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/TerminateInstancesTask.groovy

```diff
@@ -45,8 +47,14 @@ class TerminateInstancesTask extends AbstractCloudProviderAwareTask implements T
         "terminate.account.name": account,
         "terminate.region"      : stage.context.region,
         "kato.last.task.id"     : taskId,
-        "kato.task.id"          : taskId, // TODO retire this.
-        "terminate.instance.ids": stage.context.instanceIds,
+        "terminate.instances"   : stage.context.instances,
     ])
   }
+
+  static Map convert(Stage stage) {
+    def operation = [:] << stage.context
+    def instances = operation.remove("instances")
+    operation.instanceIds = instances.collect { it.id }
```

Comment:
The Orca API is changing, but I'm not changing the Clouddriver API. https://github.com/spinnaker/deck/pull/1809/files shows the call to Orca with the instance ID/launch time object. Previously, this stage passed the whole context through, which if you look at what Deck stuffed in there, the key was `instanceIds` with a list of string IDs.